src/sentry/grouping/enhancer/__init__.py

```diff
@@ -136,7 +137,10 @@ def apply_modifications_to_frame(self, frames, platform, exception_data):
             for idx, action in rule.get_matching_frame_actions(
                 match_frames, platform, exception_data, cache
             ):
-                action.apply_modifications_to_frame(frames, match_frames, idx, rule=rule)
+                with sentry_sdk.start_span(
+                    op="stacktrace_processing", name="apply_modifications_to_frame"
```

Comment:
Moving it down here from `sentry.stacktraces.processing` since we are creating too many spans and going over 1,000 spans.
<img width="535" alt="image" src="https://github.com/getsentry/sentry/assets/44410/abdee8d0-8e82-4836-82eb-5a893571c729">